cosmosdb/scale-cosmosdb-replicate-multiple-regions/scale-cosmosdb-replicate-multiple-regions.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash

```

Comment:
__RECOMMENDED__: The script generates the following warning:

```
The regionName=failoverPriority method of specifying locations is deprecated. Use --locations KEY=VALUE [KEY=VALUE ...]
to specify the regionName, failoverPriority, and isZoneRedundant properties of the location.
Multiple locations can be specified by including more than one --locations argument.
```

This should be fixed in order to avoid the script breaking in a future release of the CLI.

Reply:
Updated in upcoming commit.